tools/translation/update_po.sh

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env bash
+
```

Comment:
What is this program for?

Reply:
`/usr/bin/env` is like `which`.  `/usr/bin/env` is a "well known path" for env and will work on distros which do not have `bash` at `/usr/bin` like NixOS (ie, doing this is more portable than `/usr/bin/bash`

Reply:
lol I meant what is `update_po.sh` for

Reply:
Update the strings in `.po` files from the `.pot` file.

Reply:
Weblate does that automatically in our current workflow right? I'm not saying we shouldn't have the script but I would like to see a comment about that if is the case.

Reply:
I'm not sure Weblate does that automatically. Some months ago I noticed some strings were not in Weblate but in source, after having run that script and pushed, Weblate went in sync. Maybe that was a bug in Weblate, but I now always run `./generate_pot.sh` then `./update_po.sh` after merging translation changes from Weblate. In case it's not needed anymore it can't do harm anyway, I always run it to make sure everything is fine.

Reply:
It would be nice to have a comment about your experience with that.

Reply:
I updated the translation tool readme and placed the comment there.